meteor/sofie-server/data-access/repositories/mongo/mongo-entity-converter.ts

```diff
@@ -95,15 +108,24 @@ export class MongoEntityConverter {
 		return new Rundown({
 			id: mongoRundown._id,
 			name: mongoRundown.name,
-			isActive: false,
+			isRundownActive: false,
 			segments: [],
+			lastTimeModified: mongoRundown.modified,
```

Comment:
`lastTime` is a bit implicit since we only have one entry so this value will always be the "last time". I think `modifiedAt` would be a good candidate

Reply:
I have changed it to `modifiedAt`.